frontend/src/scenes/sessions/SessionsPlay.tsx

```diff
@@ -96,7 +96,7 @@ function _SessionsPlay(): JSX.Element {
     }
 
     return (
-        <div className="session-player">
+        <div className="session-player ph-no-capture">
```

Comment:
Not sure everything is sensitive, think we can keep stuff like the timeline, the device/resolution indicators, loaders, ...